spec/windows/interface.go

```diff
@@ -59,6 +59,8 @@ func (g *InterfaceGenerator) Generate() ([]spec.NetInterface, error) {
 			switch v := addr.(type) {
 			case *net.IPAddr:
 				ip = v.IP
+			case *net.IPNet:
```

Comment:
Should we consider range of v.Mask?

Reply:
How about replace this to `net.Parse(addr.String())` ?

Reply:
`net.ParseIP` does not accept addresses with subnet mask.